docs/synthetics/alerts.md

```diff
@@ -0,0 +1,27 @@
+title: Creating Alerts 
+description: Step by step guide for creating Synthetics Alerts
+
+Synthetics App support 2 types of alerts:
+
+* Synthetics Alerts - These alerts are raised when a run fails based on the configured conditions and rule is auto-created while creating the monitor.
```

Comment:
"....and rule..." - which rule?  Maybe alert rule?  If so, let's say "alert rule" here.

Reply:
Yes. It is alert rule. Will update